sdk/communication/Azure.Communication.Common/tests/Pipeline/HMACAuthenticationPolicyTests.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Azure.Communication.Pipeline
+{
+    public class HMACAuthenticationPolicyTests : PolicyTestBase
+    {
+        private const string secret_key = "68810419818922fb0263dd6ee4b9c56537dbad914aa7324a119fce26778a286e";
+
+        [Test]
+        public async Task TestHMACPolicyProcess()
+        {
+            var shaValue = "47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU=";
+            var authPolicy = new HMACAuthenticationPolicy(secret_key);
+            var transport = new MockTransport(new MockResponse(200));
+            await SendGetRequest(transport, authPolicy);
```

Comment:
I wonder if it makes sense to add similar methods with body that includes non English characters or the ones that would be escaped.